benchmark.py

```diff
@@ -91,43 +99,31 @@ def f1_score(prediction, ground_truth):
     num_same = sum(common.values())
     if num_same == 0:
         return 0, 0, 0
-    precision = 1.0 * num_same / len(prediction_tokens)
-    recall = 1.0 * num_same / len(ground_truth_tokens)
+    precision = num_same / len(prediction_tokens)
+    recall = num_same / len(ground_truth_tokens)
     f1 = (2 * precision * recall) / (precision + recall)
     return f1, precision, recall
 
-def agent(query , llm):
+def agent(query, llm):
     planner = TaskPlanner(autonomous=True)
-    # for autonomous = False
-    """
-    planner = TaskPlanner(autonomous=False,human_intervene=False)
-    
-    researcher = Worker(
-        role="Wikipedia Researcher",
-        instructions="You are an intelligent Wikipedia agent. Your generation should follow the example format. Finish the task if you find the answer. And you answer should be simple and straighforward. DO NOT repeat your actions.",
-        actions=[WikiSearchAction],
-        max_iterations = 3,
-    )"""
     admin = Admin(
         planner=planner,
         memory=Memory(),
         actions=[WikiSearchAction],
         llm=llm,
     )
-    # admin.assign_workers([researcher]) #un-comment this for Autonomous = False
-
     res = admin.run(
         query=query,
-        description="Provide answer for the query.You should decompose your task into executable actions.",
+        description="Provide answer for the query. You should decompose your task into executable actions.",
     )
     return res
 
-def run_agent(level = 'easy'):
-    os.environ["AZURE_BASE_URL"] = ""
-    os.environ["AZURE_DEPLOYMENT_NAME"] = ""
-    os.environ["AZURE_MODEL_NAME"]="gpt4"
-    os.environ["AZURE_OPENAI_API_VERSION"]=""
-    os.environ["AZURE_OPENAI_API_KEY"]=  ""
+def run_agent(level='easy'):
```

Comment:
revert the changes for this function

Reply:
I have reverted. Hopefully....Any other problems let me know. Thanks for taking the time with me on this.